tests/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
Just double-checked, this file is empty (i.e. no whitespace). I'm guessing github just shows the above since the file was created. And yes, I am using the whitespace add-on.